src/modules/service-catalog/ServiceCatalogItemPage.tsx

```diff
@@ -147,47 +107,37 @@ export function ServiceCatalogItemPage({
       ? organizations[0]?.id?.toString()
       : null;
 
-  const toggleDescription = () => {
-    setIsExpanded(!isExpanded);
-  };
-
   return (
     <Container>
-      <LeftColumn>
-        <DescriptionWrapper>
-          <ItemTitle tag="h1">{serviceCatalogItem.name}</ItemTitle>
-          <CollapsibleDescription expanded={isExpanded}>
-            {serviceCatalogItem.description}
-          </CollapsibleDescription>
-          <ToggleButton isLink onClick={toggleDescription}>
-            {isExpanded
-              ? t("service-catalog.item.read-less", "Read less")
-              : t("service-catalog.item.read-more", "Read more")}
-            <Button.EndIcon>
-              {isExpanded ? <ChevronUp /> : <ChevronDown />}
-            </Button.EndIcon>
-          </ToggleButton>
-        </DescriptionWrapper>
-        <FromFieldsWrapper>
-          <ItemRequestForm
-            requestFields={requestFields}
-            baseLocale={baseLocale}
-            hasAtMentions={hasAtMentions}
-            userRole={userRole}
-            userId={userId}
-            brandId={brandId}
-            defaultOrganizationId={defaultOrganizationId}
-            handleChange={handleChange}
-          />
-        </FromFieldsWrapper>
-      </LeftColumn>
-      <RightColumn>
-        <ButtonWrapper>
-          <Button isPrimary size="large" isStretched>
-            {t("service-catalog.item.submit-button", "Submit request")}
-          </Button>
-        </ButtonWrapper>
-      </RightColumn>
+      {serviceCatalogItem && (
+        <>
```

Comment:
Do we always need to render the container and then render the content with an empty fragment (`<>`) as a wrapper? Can we do `return serviceCatalogItem ? <Container>...</Container> : null` or do we need the container for some reason?

Reply:
Yes we can do this, I changed that.